static/moodytunes/js/browse.js

```diff
@@ -142,7 +152,7 @@
         var genre = document.getElementById('id_genre').value || undefined;
 
         document.MoodyTunesClient.getBrowsePlaylist(
-            emotion, jitter, limit, genre, context, displayBrowsePlaylist
+            emotion, jitter, limit, genre, context, false, displayBrowsePlaylist
```

Comment:
Remove this `false` param after removing the argument in the `gerBrowsePlaylist` function